File: mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int s = 0;
    int b = 0;
    int a;
    do
    {
        a = get_int("Digite a altura da piramide que deseja(aceitavel entre 1 e 8)");
    }
    while (a < 1 || a > 8);
        for (int i = 0; i < a; i++)
            {
                for (int j = 7; j > i; j--)
                {
                        printf(" ");
                }
                for (int h = -1; h < i; h++)
                {
                        printf("#");
                }
                printf(" ");
                for (int h = -1; h < i; h++)
                {
                        printf("#");
                }
                for (int j = 7; j > i; j--)
                {
                        printf(" ");
                }
            printf("\n");
            }
}

File: python/mario.py
from cs50 import get_int
# solicita a altura da piramide (tem que ser de 1 a 8)
altura = get_int ("Altura (1 a 8) ?")
if altura < 1 or altura > 8:
    print("Numero Invalido")
if altura > 1 and altura < 8:
    for i in range(altura - (altura - 1)):
        print(" ", end="")
    for j in range(altura):
        print("#", end="")